tooling/cli/src/info/env_system.rs

```diff
@@ -30,23 +58,24 @@ fn build_tools_version() -> crate::Result<Option<Vec<String>>> {
       let _ = file.write_all(VSWHERE);
     }
   }
+
+  // Check if there are Visual Studio installations that have the "MSVC - C++ Buildtools" component
   let output = Command::new(vswhere)
     .args([
       "-prerelease",
       "-products",
       "*",
-      "-requiresAny",
       "-requires",
-      "Microsoft.VisualStudio.Workload.NativeDesktop",
-      "-requires",
-      "Microsoft.VisualStudio.Workload.VCTools",
+      "Microsoft.VisualStudio.Component.VC.Tools.x86.x64",
       "-format",
       "json",
     ])
     .output()?;
-  Ok(if output.status.success() {
+
+  Ok(if output.status.success() && has_windows_sdk_libs() {
```

Comment:
maybe separate this check into its own item with its own error message?

Reply:
`unwrap_or_default` is fine here since we are only looking for installed instances of visual studio, and we only print the error if we couldn't find any.

`has_windows_sdk_libs` on the other hand, could be `false` while vs is installed and so it should be its own section item and have its own error message